pyworklistserver/user_config.py

```diff
@@ -0,0 +1,51 @@
+""" Sets variables for use of the worklist-generators"""
+import random
+
+#Maximall number of total patient-objects delivered from generator.
+maxAmountOfWorklistExams = 70
+
+#Minimum number of total patient-objects delivered from generator.
+minAmountOfWorklistExams = 20
+
+#Rate of clean patient-objects delivered from generator.
```

Comment:
The term 'rate' is a bit unclear hear. Rate is a proportion/fraction with a numerator and a denominator. I would therefore expect a number between 0 and 1 here. Do you need to specify both rate of clean and random exams? If you choose a likelihood of clean exams of 0.3, should't there always be a likelihood of 0.7 for getting a random exam?